src/dotnet/commands/dotnet-tool/update/ToolUpdateCommand.cs

```diff
@@ -112,8 +124,11 @@ public override int Execute()
                         new PackageLocation(nugetConfig: GetConfigFile(), additionalFeeds: _additionalFeeds),
                         packageId: _packageId,
                         targetFramework: _framework,
+                        versionRange: versionRange,
                         verbosity: _verbosity);
 
+                    EnsureVersionIsHigher(oldPackageNullable, newInstalledPackage);
```

Comment:
Additionally, can this check be done prior to entering the transaction scope to install the package?  NuGet won't install a version older than what's requested, so do we need to install the package to ensure the version is newer than the version we already have?

Reply:
Scratch that, I forgot we can take version ranges.  Resolving this conversation.